modules/shared/components/molecules/UploadingImage/UploadingImage.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import Image from 'next/image';
+import type { IUploadingImage } from './IUploadingImage';
+import styles from './UploadingImage.module.css';
+import TextInput from '../../atoms/TextInputs/TextInput';
+import * as ETextInput from '../../atoms/TextInputs/types/ETextInput';
+import VerticalThreeDots from '../../icons/verticalThreeDots.svg';
+
+const UploadingImage: FC<IUploadingImage.IProps> = ({
+  imageUrl = 'https://source.unsplash.com/random',
+  handleVerticalThreeDotsClick,
+  textInputLetter,
+  textInputId,
+}): ReactElement => {
+  return (
+    <div className={styles.container}>
+      <div className="relative">
+        <Image
+          src={imageUrl}
+          layout="fill"
+          objectFit="contain"
+          className="w-full h-full rounded-t-md mb-1 "
+        />
+        <button
+          type="button"
+          onClick={handleVerticalThreeDotsClick}
+          className={styles.button}
+        >
+          <VerticalThreeDots className="fill-dark-grey" />
+        </button>
+      </div>
+      <TextInput
```

Comment:
what will this text field do and how it will pass its value to the desired parent State? where is its onChange or onBlur handler?

Reply:
again valid point, i thought about it, but i was not sure yet how we gonna use react hook form to satisfy this input, so i decided to keep it like that till i am fully aware of it, then i will modify it, but i think i have to figure it now.